package.json

```diff
@@ -5,12 +5,10 @@
   "directories": {
     "test": "test"
   },
-  "main": "lib",
+  "main": "lib/index.js",
```

Comment:
what's the need for this change?

Reply:
no need beyond we're starting to make these values consistent and this is how we do it elsewhere

Reply:
cool, feel free to resolve the remaining open items :-)